PubnativeLite/PubnativeLite/Banner/VWAdvertView.h

```diff
@@ -0,0 +1,56 @@
+//
+//  Copyright © 2020 PubNative. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+#import <CoreLocation/CoreLocation.h>
+#import "VWAdRequest.h"
+#import "HyBidAdSize.h"
+#import "HyBidAdView.h"
+
+@class VWAdvertView;
+
+@protocol VWAdvertViewDelegate
+
+- (void)advertViewDidReceiveAd:(nonnull VWAdvertView *)adView;
+
+- (void)advertView:(nonnull VWAdvertView *)adView didFailToReceiveAdWithError:(nullable NSError *)error;
+
+@end
+
+@interface VWAdvertView : HyBidAdView<HyBidAdViewDelegate>
+
+@property (nonatomic, weak, nullable) id <VWAdvertViewDelegate, NSObject> delegateVerve;
```

Comment:
If this is the only change (`delegete` -> `delegateVerve` [or `verveDelegate` could be better maybe 🤔 ]) that publisher has to make, then i think we are okay with this. What do you think @eros902002 ?

Reply:
If we have no choice then yes. Although it would be a bit confusing to them but should be fine. The issue is that the parent class already has a variable named delegate, right?

Reply:
Yes @eros902002. I think the issue arises from VWAdvertView is a UIView afterall. The solution that we have in Interstitial is not working here fully. Maybe @faresbh can explain it a bit more detailed. So that you can have an idea what's going on or even better an advice 😍

Reply:
VWAdvertview now has a property for Hybidadview so we will not use a special delegate.